pkg/claimsstore/rootcommitpersister.go

```diff
@@ -47,3 +47,12 @@ func (p *RootCommitsPGPersister) Get(rootHash string) (*RootCommit, error) {
 	}
 	return rootCommit, nil
 }
+
+// GetLatest returns the most recent root committed to the tree
+func (p *RootCommitsPGPersister) GetLatest() (*RootCommit, error) {
+	rootCommit := &RootCommit{}
+	if err := p.db.Last(rootCommit).Error; err != nil {
```

Comment:
Since the PK for this table is the root, which I'm guessing is a hash, will finding the last value be correct?  I'm assuming it is returning the last value based on some sort order, but I could be wrong.  I see your tests below have a clear ordering to the hashes.

Reply:
It's based on created_date, but now I think this is wrong.  Should sort by block number